lib/matplotlib/widgets.py

```diff
@@ -3278,10 +3290,23 @@ def _press(self, event):
         self._rotation_on_press = self._rotation
         self._set_aspect_ratio_correction()
 
+        match self._get_action():
+            case _RectangleSelectorAction.ROTATE:
+                # TODO: set to a rotate cursor if possible?
+                pass
+            case _RectangleSelectorAction.MOVE:
+                self._set_cursor(backend_tools.cursors.MOVE)
+            case _RectangleSelectorAction.RESIZE:
+                # TODO: set to a resize cursor if possible?
+                pass
```

Comment:
Do you need diagonal resize here, or something different? Resize usually needs two directions (or four if you're counting straight and diagonal together.)